docs/api/portable-stories-jest.md

```diff
@@ -245,12 +245,16 @@ The story is prepared by running [`composeStories`](#composestories) or [`compos
 
 ### 3. Load
 
-**(optional)**
-
-Stories can prepare data they need (e.g. setting up some mocks or fetching data) before rendering by defining [loaders](../writing-stories/loaders.md). In portable stories, the loaders are not applied automatically—you have to apply them yourself.
+Stories can prepare data they need (e.g. setting up some mocks or fetching data) before rendering by defining [loaders](../writing-stories/loaders.md) or [beforeEach](../writing-stories/mocking-modules.md#setting-up-and-cleaning-up). In portable stories, loaders and beforeEach are not applied automatically — you have to apply them yourself.
```

Comment:
The link to beforeEach is based on this: https://github.com/storybookjs/storybook/pull/26858/files#diff-82d07127279d7f0e9cbf1f0735ac589416a2ca7ffc3d5ff961b357d3a4a6c803R154

Maybe there's a better link

Reply:
There's also this one, which may be more appropriate for the portable stories case:

```suggestion
Stories can prepare data they need (e.g. setting up some mocks or fetching data) before rendering by defining [loaders](../writing-stories/loaders.md) or [beforeEach](../writing-tests/interaction-testing.md#run-code-before-each-test). In portable stories, loaders and beforeEach are not applied automatically — you have to apply them yourself.
```